test/support/CustomMatchers.scala

```diff
@@ -432,6 +437,20 @@ object CustomMatchers {
 
```

Comment:
```
  def haveCustodialHistory(expectedValue: String): Matcher[Document] = (document: Document) =>
    singleValueMatcher(
      label = "custodial history",
      expectedValue = expectedValue,
      actualValue = document.select("#custodial-history").text()
    )

  def haveErrorMessageForCustodialHistory(expectedValue: String): Matcher[Document] =
    haveErrorMessageForField("custodial-history", "#custodial-history-error", expectedValue)

```
@felixgb does this need to be replaced by FieldNames.custodialHistory, appears it might have been missed